install/helm/README.md

```diff
@@ -27,6 +27,26 @@ The command deploys Agones on the Kubernetes cluster with the default configurat
 
 > **Tip**: List all releases using `helm list`
 
+## Namespaces
+
+By default Agones is configured to work with game servers deployed in the default namespace. If you are planning to use other(s) namespace(s) you can configure Agones via the parameter `gameservers.namespaces`.
```

Comment:
> with game servers deployed in the `default` namespace

May want to add backticks around "default"

Reply:
> If you are planning to use other namespaces

I'd suggest dropping the (s) on both, I don't think they are needed, and I feel like it reads a bit better.